### ISSUE TITLE
Implement `--no-style` to disable just style suggestions

### ISSUE THREAD

**Issue by certik** [lfortran/lfortran#564]
*Original issue*: https://gitlab.com/lfortran/lfortran/-/issues/733

We have `--no-warnings` to disable all warnings, and we should also have `--no-style-warnings` (or `--no-style-suggestions` or just `--no-style`) to only disable style suggestions.

**Comment by kargl** [comment]
It would be really, really, really, nice to have this option.  Compiling old code is swamped with unhelpful style hints.

**Comment by certik** [comment]
I agree, I am sorry we didn't implemented yet. For now just use `--no-warnings`, we currently only emit very few warnings anyway.
